### PR TITLE
Changed layout of signature box

### PR DESCRIPTION
**Pull request by ryanguybjss**
## Description and relevant Jira numbers

* Moved the "Sign Here" prompt and the retry button in the signature area to above the area itself to allow room for Welsh translations

* Moved the "Enter a Signature" validation message into signature-area component

* Stopped signature-area from checking the isvalid field as it does nothing

* Extended the signature line and placed it in the correct location

* Made the validation bar the correct size

* Applied the changes to signature-area components on both the health-declaration page and the waiting-room page

* Changed the padding around the signature areas

* Deleted the now redundant 'sign-here.png'

image
image

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [ ] PR link added to JIRA
- [ ] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

### CODE REVIEW

File: src/pages/waiting-room/waiting-room.html [hunk 1]
```diff
@@ -81,23 +81,30 @@ <h4>{{ 'waitingRoom.declarationIntent' | translate}}:</h4>
...
-          </ion-row>
-          <ion-row class="declaration-signature-row">
-            <ion-col class="declaration-signature-col">
+            </ion-row>
+            <ion-row class="declaration-signature-row">
+              <ion-col class="declaration-signature-col">
               <signature-area formControlName="signatureAreaCtrl" [signature]="pageState.signature$ | async"
                 retryButtonText="{{ 'waitingRoom.signatureRetry' | translate }}"
-                signHereText="{{ 'waitingRoom.signaturePrompt' | translate }}">
+                signHereText=" ">
```
Comment (TomBillingtonUK) [inline src/pages/waiting-room/waiting-room.html]: Are we doing this on the health declaration page as well for consistency 

Also not happy with the signHereText being an empty string can we remove it and if it's not used in either signature box use. Remove it from the component.

File: src/pages/waiting-room/waiting-room.html [hunk 2]
```diff
@@ -81,23 +81,30 @@ <h4>{{ 'waitingRoom.declarationIntent' | translate}}:</h4>
           </ion-row>
         </ion-col>
       </ion-row>
+
```
Comment (TomBillingtonUK) [inline src/pages/waiting-room/waiting-room.html]: blank line can we remove

**Comment by jameshighfield** [comment]
Can we remove the arrow where the text was previously to match the design? I'm not sure it makes sense to keep it without the text.

Design:

3 1 Waiting Room -2

**Comment by jameshighfield** [comment]
Changes look good. Please can you upload a screenshot of how it looks now?

File: src/components/signature-area/signature-area.html [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+        <ion-col>
+            <button (click)="clear()" class="retry-button" [disabled]="!signature">
+                <div id="retry-icon" class="retry-icon" [ngStyle]="{'background-image':'url(' + retryImage + ')'}">
+                </div>
+                <label id="retry-button-label"
+                    class="sign-here-text">{{retryButtonText? retryButtonText : "Retry"}}</label>
+            </button>
+        </ion-col>
+    </ion-row>
+    <div>
```
Comment (ghost) [inline src/components/signature-area/signature-area.html]: This `div` can probably be removed, just have two `ion-row`s as siblings.

File: src/components/signature-area/signature-area.ts [hunk 4]
```diff
@@ -35,6 +35,12 @@ export class SignatureAreaComponent implements ControlValueAccessor {
   @Input()
   public signHereText: string;
 
+  @Input()
+  public isValidText: string;
```
Comment (ghost) [inline src/components/signature-area/signature-area.ts]: I'd change the name of this `@Input` to something along the lines of `validationErrorText`.

**Review by ghost — APPROVED** [review]
Couple of minor suggestions, but looks good. 👍

**Review by TomBillingtonUK — APPROVED** [review]
Looks good :)

**Pull request merged**